cds_ils/importer/documents/importer.py

```diff
@@ -198,9 +198,27 @@ def update_document(self, matched_document):
             click.secho(e.original_exception.message, fg="red")
             db.session.rollback()
 
+    def validate_found_matches(self, not_validated_matches):
+        matches = []
+        partial_matches = []
+        document_class = current_app_ils.document_record_cls
+        title = self.json_data.get("title")
+
+        for pid in not_validated_matches:
+            document = document_class.get_record_by_pid(pid)
+            document_title = document['title']
+
+            if document_title == title:
+                matches.append(document['pid'])
+            else:
+                partial_matches.append(document['pid'])
+
+        return matches, partial_matches
```

Comment:
where are the partial matched displayed or consumed ?

Reply:
for now nowhere 😅